spec/features/consumer/shopping/shopping_spec.rb

```diff
@@ -50,6 +50,14 @@
         page.should have_selector "option[selected]", text: 'turtles'
       end
 
+      it "displays additional notes for order cycle" do
+        exchange1.update_attribute :pickup_time, "hedgehog"
+        exchange1.update_attribute :pickup_instructions, "Tickle my belly before you pick me up"
+        visit shop_path
+        select "hedgehog", :from => "order_cycle_id"
+        page.should have_content exchange1.pickup_instructions
```

Comment:
@RohanM Rebased, split and resolved conflict. 

I'm aware of TDD, I just don't think I'm quite there yet. I'm overcoming my specophobia and I've written some good tests recently, which is good enough for now...